<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="select_option_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(29, 155, 240)</string>
   </property>
   <property name="text">
    <string>SELECT OPTION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_img_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color:rgb(29, 155, 240);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Upload Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="video_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color:rgb(29, 155, 240);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Live Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_back">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
